app/src/main/java/com/android/fundamentals/workshop02/Workshop2ViewModel.kt

```diff
@@ -9,24 +9,27 @@ class Workshop2ViewModel(
     private val interactor: LoginInteractor
 ) : ViewModel() {
 
-    //TODO 06: Create private MutableLiveData for state storing. Set initial state State.Init
+    //TODO 06: Create private property MutableLiveData for state storing.
+    // Pass initial value "State.Init" in constructor: MutableLiveData(State.Init()).
 
-    //TODO 07: Create public LiveData (only getter) for provide state outside. Init with your private MutableLiveData
+    //TODO 07: Create public property LiveData as a getter to provide state outside.
+    // Init this public getter with your private MutableLiveData.
 
     fun login(userName: String, password: String) {
         viewModelScope.launch {
-            //TODO 08: Set loading state to private liveData
+            //TODO 08: Set "State.Loading()" to the private liveData's value.
 
             val loginResult = interactor.login(userName = userName, password = password)
 
-            //TODO 09: Handle loginResult and create new state depend on it (success or error)
+            //TODO 09: Handle "loginResult" with "when()".
+            // Set actual "State. ...()" to the private liveData's value (success, errors).
 
-            //TODO 10: Set new state to private liveData
+            //TODO 10: Set "State.Default()" to the private liveData's value.
```

Comment:
looks like
//TODO 10: Set "State.Default()" to the private liveData's value.
non needed